recipes/PySimpleGUIWx/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pysimpleguiwx" %}
+{% set version = "0.15.0" %}
+{% set styledName = "PySimpleGUIWx" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ styledName }}-{{ version }}.tar.gz
+  sha256: "3432d71f460d71083590422dc0f6356f0b426701715c0af0e9677d9a60e0a3b6"
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - wxpython
```

Comment:
Are you sure `wxpython` should be in the `host` section and not `run`? Usually only `pip` and `python` belong in the `host` section, and all other dependencies in `run`.